.travis.yml

```diff
@@ -1,61 +1,20 @@
-sudo: required
-dist: trusty
-
+dist: bionic
 language: go
-go_import_path: github.com/coreos/ignition
 
 go:
-  - "1.11"
-  - "1.12"
+  - "1.11.x"
+  - "1.13.x"
+
+arch:
+  - arm64
+  - amd64
 
-env:
-  matrix:
-   - TARGET=amd64
-   - TARGET=arm64 GIMME_ARCH=arm64 GIMME_CGO_ENABLED=1
+install: echo nop
```

Comment:
Reading https://docs.travis-ci.com/user/job-lifecycle/#skipping-the-installation-phase, maybe `install: skip`?